.github/workflows/cicd_develop_staging.yaml

```diff
@@ -36,7 +36,7 @@ env:
   #
   #######################################
   STG_HOLOGRAPH_INDEXER_HELM_CHART_VERSION: 0.0.69
-  INDEXER_RELEASE_NAME: indexer-dev  # format -> [release_name]-indexer-[env]
+  INDEXER_RELEASE_NAME: indexer-dev # format -> [release_name]-indexer-[env]
```

Comment:
@alexanderattar why are these files modified? I believe whatever is in develop should be the correct version.

Reply:
this was just a whitespace issue caught by prettier. the change shouldn't effect anything since it's just a comment

Reply:
ah ok. I see you are right about that.